bundles/org.openhab.binding.telegram/README.md

```diff
@@ -52,6 +52,29 @@ Please note that the things cannot be used to send messages. In order to send a
 | `chatIds`               |         | Yes      | Comma-separated list of chat ids                                                             |
 | `botToken`              |         | Yes      | authentication token                                                                         |
 | `parseMode`             |  None   | No       | Support for formatted messages, values: Markdown or HTML.                                    |
+| `proxyHost`             |  None   | No       | Proxy host for telegram binding, it doesn't affect entire openhab.                           |
```

Comment:
IMHO it's not necessary to tell the user that the host configuration for a THING doesn't affect the OH config.

Reply:
> IMHO it's not necessary to tell the user that the host configuration for a THING doesn't affect the OH config.

fixed